.github/ISSUE_TEMPLATE/discussion.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Discussion
+about: Start a discussion topic, or brainstorm an new idea, related to WinUI
+title: 'Discussion:'
+labels: discussion
+assignees: ''
+
+---
+
+<!-- This is a template for discussion issues.
+These issues invite members of our community to discuss any topics or ideas related to WinUI.  For example, you might
```

Comment:
```suggestion
Discussion issues can be about any topics or ideas related to WinUI.  For example, you might
```
First sentence sounds like it's explaining why we created it and not why/how to use it - maybe make it more opener-oriented?